src/python/pants/backend/jvm/tasks/jvm_compile/rsc/rsc_compile.py

```diff
@@ -40,7 +44,12 @@
 
 def fast_relpath_collection(collection):
   buildroot = get_buildroot()
-  return [fast_relpath(c, buildroot) for c in collection]
+  def f(c, buildroot):
+    try:
+      return fast_relpath(c, buildroot)
```

Comment:
Prefer `fast_relpath_optional` to catching.

Reply:
Nice. Thanks.